### DISCUSSION TITLE
Coordinator error

### DISCUSSION THREAD

**Discussion by jim-counter** [subquery/subql#944] [Frontier TestNet - Bug Reporting]
Saw this in the logs this morning. From the message it looks like a downstream request failed? Things look OK so I'm guessing the system has recovered (on the next attempt to update the network?)

`coordinator_service  | 2022-04-28T06:38:34.811Z <network> ERROR failed to update network: Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (error={"reason":"missing response","code":"SERVER_ERROR","requestBody":"{\"method\":\"eth_call\",\"params\":[{\"from\":\"0x7a6b4344bbd5e1af749d8d0a61812eee46631e6d\",\"to\":\"0xa500a3ad54223b7e18d7d73da34f2ef06aca77ec\",\"data\":\"0x1f6fbdf600000000000000000000000082aada6f7adedc1b68bdfa40d09581d6bb65ef76\"},\"latest\"],\"id\":10209,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","serverError":{"errno":-3001,"code":"EAI_AGAIN","syscall":"getaddrinfo","hostname":"sqtn.api.onfinality.io"},"url":"https://sqtn.api.onfinality.io/public"}, data="0x", code=CALL_EXCEPTION, version=providers/5.6.4)`

**Comment by mzxyz** [comment]
Thanks for reporting this issue, this kind of error issue should be resolved in the latest `onfinality/subql-coordinator@v0.1.8`
